Documentation/CRDs/Cluster/external-cluster/upgrade-external.md

```diff
@@ -15,6 +15,7 @@ Upgrading the cluster would be different for restricted caps and non-restricted
 2. If the consumer cluster has restricted caps
 
     Restricted users created using `--restricted-auth-permission` flag need to pass mandatory flags: '`--rbd-data-pool-name`(if it is a rbd user), `--k8s-cluster-name` and `--run-as-user`' flags while upgrading, in case of cephfs users if you have passed `--cephfs-filesystem-name` flag while creating CSI users then while upgrading it will be mandatory too. In this example the user would be `client.csi-rbd-node-rookstorage-replicapool` (following the pattern `csi-user-clusterName-poolName`)
+    If --cephx-key-rotation was set, it adds `-rotated` suffix to the user name, for example: `client.csi-rbd-node-rookstorage-replicapool-rotated`
```

Comment:
we are not rotating the key here, just taking a prefix for the csi user. should we call it as prefix?

couple of things
* This will not change anything if the user already exists (if the script runs multiple time)
* Document the steps or provide a command to remove the old users as well?